modules/migrating-to-multi-arch-cli.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * updating/migrating-to-multi-payload.adoc
+
+:_content-type: PROCEDURE
+[id="migrating-to-multi-arch-cli_{context}"]
+= Migrating a cluster to a heterogenous payload using the CLI
+
+.Prerequisites
+
+* You must be logged in as a user with `cluster-admin` privileges on all clusters.
+* You must be updated to {product-title} version 4.13.0 or higher. For more information on how to update your cluster version, see _Updating a cluster using the web console_ or _Updating a cluster using the CLI_ listed in the Additional resources section.
+
+.Procedure
+
+. Install your {product-title} cluster on your preferred platform. For more information on selecting a supported platform for your cluster installation, see _Selecting a cluster installation type_ in the Additional resources section.
+. Set the channel to `stable-4.13`, `fast-4.13`, or `eus-4.13` with `oc adm upgrade channel <channel>`. For more information about channels, refer to _Understanding update channels and releases_ listed in the Additional resources section.
++
+[source,terminal]
+----
+$ oc adm upgrade channel <channel>
+----
+
+. Upgrade to multi-arch with the `oc adm upgrade --to-multi-arch` command:
```

Comment:
> Upgrade
It's not that accurate to mention `upgrade` here to me. It's actually a arch migration between the same version rather than upgrade to a higher version.

Reply:
replacing with "migrate" :)